tools/osv-linter/internal/checks/affected.go

```diff
@@ -0,0 +1,21 @@
+package checks
+
+import (
+	"github.com/tidwall/gjson"
+)
+
+var CheckAffectedFieldValid = &CheckDef{
+	Code:        "A0001",
+	Name:        "affected-field-valid",
+	Description: "affected field validates",
```

Comment:
The style I've been going with is more like a test assertion

```suggestion
	Description: "every record has affected data",
```